src/View/View_PDF.php

```diff
@@ -631,8 +631,16 @@ public function get_core_template_styles( $settings, $entry ) {
 	 *
 	 * @since 4.0
 	 */
-	public function load_core_template_styles( $settings ) {
-		return $this->load( 'core_template_styles', [ 'settings' => $settings ], false );
+	public function load_core_template_styles( $settings, $form, $entry ) {
```

Comment:
For backwards compatibility, let's make $form and $entry optional parameters and set to an empty array by default. This will prevent any backwards compatible issues for any one directly using this method.